configs/config.development.json

```diff
@@ -0,0 +1,26 @@
+{
+    "socket": 8080,
+
+    "devAssetServer": true,
+
+    "cluster": {
+        "workers": 1,
+        "timeout": 1
+    },
+
+    "hosts": {
+        "static": "/"
+    },
+
+    "i18n": {
+        "langs": ["en", "ru"]
+    },
+
+    "supportedBrowsers": [
```

Comment:
I believe `i18n` may stay in a config because a case when we need different lists on locales for different instances of an app is highly probable.

`supportedBrowsers` IMO should be moved into `.enb/make.js` if isn't used anywhere else.